Problem with avr/pgmspace.h

I'm trying to compile some code with PlatformIO and I'm getting this error:

Compiling .pioenvs/nodemcuv2/src/EnviroMonitorStation.ino.o
Compiling .pioenvs/nodemcuv2/src/EnviroMonitorStation.ino.o
Compiling .pioenvs/nodemcuv2/lib/ESP8266wifi_ID1101/ESP8266wifi.o
Compiling .pioenvs/nodemcuv2/lib/ESP8266wifi_ID1101/ESP8266wifi.o
In file included from .piolibdeps/ESP8266wifi_ID1101/ESP8266wifi.cpp:9:0:
.piolibdeps/ESP8266wifi_ID1101/ESP8266wifi.h:23:26: fatal error: avr/pgmspace.h: No such file or directory
#include <avr/pgmspace.h>
^
compilation terminated.

I found this https://github.com/esp8266/Arduino/issues/2366 

I think it might be related. Is there anything I could do to allow compilation?

--- a/ESP8266wifi.h
+++ b/ESP8266wifi.h
@@ -21,12 +21,11 @@
 
 #include <inttypes.h>
 
-#if defined ( ESP8266 )
-  #include <pgmspace.h>
+#if defined(ESP8266)
+#include <pgmspace.h>
 #else
-  #include <avr/pgmspace.h>
+#include <avr/pgmspace.h>
 #endif
-
 
 #include "HardwareSerial.h"
 

--- a/ESP8266wifi.h
+++ b/ESP8266wifi.h
@@ -20,7 +20,14 @@
 #endif
 
 #include <inttypes.h>
-#include <avr/pgmspace.h>
+
+#if defined ( ESP8266 )
+  #include <pgmspace.h>
+#else
+  #include <avr/pgmspace.h>
+#endif
+
+
 #include "HardwareSerial.h"
 
 #define SERVER '4'

